scripts/presubmit.sh

```diff
@@ -86,6 +86,9 @@ if [ -n "${OUT}" ]; then
   exit 1
 fi
 
+while true; do docker ps >>"${ARTIFACTS}/docker.out"; done &
```

Comment:
Can we make this per-run (like is there a build ID or something we can use)?

Reply:
This actually saves to a sub-directory of where the job log is located, it's pretty well separated from run to run